git-link.el

```diff
@@ -514,7 +514,9 @@ return (FILENAME . REVISION) otherwise nil."
   (message (replace-regexp-in-string "%" "%%" link t t))
   (setq deactivate-mark t)
   (when git-link-open-in-browser
-    (browse-url link)))
+    (unless (fboundp git-link-open-in-browser)
+      (setq git-link-open-in-browser 'browse-url))
```

Comment:
Why set this, seems better to let it remain `t`? Is this because `defcustom`'s type changed?

Reply:
This allows to be compatible with existing configs. If the value is `t` then `browse-url` will be used. Otherwise configured browser will be used.

Reply:
Thanks. 

We can also do that without overwriting users' value: 

```el
(when git-link-open-in-browser
  (if (fboundp git-link-open-in-browser)
      (funcall git-link-open-in-browser link)
    (browse-url link)))
```

Not a fan of overwriting global values. Plus user may have some logic built on it that we then break.

Reply:
This is indeed better.
I thought user may see the change in `customize-group` and be informed about the new feature. It may not be necessary at all. That is why I overwrote its value.

Reply:
Should I change it?

Reply:
> Do people actually use Customize :)

I am using it, but I also use the laptop's DVD drive :rofl: 
I made the change. Please see my other reply about the `'function`.

Reply:
> I am using it, but I also use the laptop's DVD drive 🤣

💽